t/PaymentsAdvanced/Response/SecureToken.t

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env perl;
```

Comment:
This should be needed. The `perlcriticrc` file needed to be updated, but now that you rebased against my changes, it should work fine without.